src/parser/jobs/pct/modules/Weaving.ts

```diff
@@ -15,6 +15,9 @@ export default class Weaving extends CoreWeaving {
 		this.data.actions.RAINBOW_DRIP.id,
 	]
 
+	// Star Prism's cure isn't a real action, it can't hurt you
+	override ignoredActionIds = [this.data.actions.STAR_PRISM_CURE.id]
```

Comment:
man it isn't even an fflogsism, it's actually in the game data. thanks. i hate it.